src/test/atlas_planned_maintenance_testing/mod.rs

```diff
@@ -1,3 +1,7 @@
+// NOTE: Please run a patch against drivers-atlas-testing when making changes within this file. The
+// DRIVER_REPOSITORY and DRIVER_REVISION fields for the Rust axis in drivers-atlas-testing's
+// evergreen config file can be updated to test against your branch.
```

Comment:
i think this would be good practice going forward, otherwise it's difficult to detect when changes here break the planned maintenance tests